src/main/java/com/solvd/pages/mobile/pim/EmployeePageMobile.java

```diff
@@ -38,8 +39,14 @@ public void clickMenuButtonByHref(String href) {
         leftMenu.getButtonByHref(href).click();
     }
 
-    private String getValueFromElement(WebElement element) {
+    @Override
+    public void getMenuByClick() {
+        menuButton.click();
+        waitUntil(ExpectedConditions.visibilityOfElementLocated(leftMenu.getBy()), 30);
+    }
+
+    private String getValueWrapper(ExtendedWebElement element) {
         JavascriptExecutor js = (JavascriptExecutor) this.getDriver();
-        return (String) js.executeScript("return arguments[0].value;", element);
+        return (String) js.executeScript("return arguments[0].value;", element.getElement());
```

Comment:
Wouldn't be simpler to just use `element.getAttribute("value")` instead of `JavascriptExecutor`?

Reply:
Nah, tried it. Somehow it does not work for mobile. It works fine in Desktop and there you can see it, but, it doesn't work for mobile.